cmd/manager/main.go

```diff
@@ -49,52 +50,76 @@ var (
 )
 
 func init() {
-	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+	if err := clientgoscheme.AddToScheme(scheme); err != nil {
+		panic(err)
+	}
 
-	podNamespace, ok := os.LookupEnv("POD_NAMESPACE")
-	if !ok {
-		panic("POD_NAMESPACE is not set")
+	//+kubebuilder:scaffold:scheme
+}
+
+func main() {
+	if err := cmd().Execute(); err != nil {
+		setupLog.Error(err, "error running manager")
+		os.Exit(1)
 	}
-	namespace = podNamespace
+}
 
-	k, ok := os.LookupEnv("NGROK_API_KEY")
-	if !ok {
-		panic("NGROK_API_KEY is not set")
+type managerOpts struct {
```

Comment:
flags/config got moved into this struct, which feels slightly cleaner. :man_shrugging:

Reply:
Nice! I _think_ long term we want to move these docs to this https://book.kubebuilder.io/component-config-tutorial/tutorial.html based on what they say in their docs, but i haven't dug into it much. This is better for now though

Reply:
Yup, indeed.